packages/core/addon/templates/components/visualization-config/line-chart.hbs

```diff
@@ -1,8 +1,39 @@
-{{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<span class="line-chart-config__header">Chart Style Options</span>
+<div class="line-chart-config__style-options">
+  {{#if (eq type "line-chart")}}
+    <div class="line-chart-config__curve-opt">
+      <span class="line-chart-config__curve-opt-title">Line Style</span>
+      {{#power-select 
+        triggerClass="line-chart-config__curve-opt-select"
+        options=curveOptions 
+        selected=(readonly options.type.curve)
+        onchange=(action "onUpdateType" "curve")
+        searchEnabled=false
+        placeholder="line"
```

Comment:
Do we need this if we have a `selected` value?

Reply:
Yeah, so it displays the default value if there is no selection.